pkg/cli/client.go

```diff
@@ -290,6 +294,22 @@ func (p *requestsPool) doJob(ctx context.Context, job *manifestJob) {
 		)
 	}
 
+	isSigned := false
+	cosignTag := strings.Replace(digestStr, ":", "-", 1) + "." + remote.SignatureTagSuffix
+	resp, _ := resty.R().Head(*job.config.servURL + "/v2/" + job.imageName + "/manifests/" + cosignTag)
```

Comment:
check for error and handle it. same for the notation signature

Reply:
I added the error check

Reply:
I don' t see it yet :D

don't ignore resty errors.

Reply:
I forgot to push the update, I'm sorry! It should be fine now.